src/algebra/gcd_monoid.lean

```diff
@@ -94,19 +128,20 @@ theorem dvd_antisymm_of_normalize_eq {a b : α}
   a = b :=
 ha ▸ hb ▸ normalize_eq_normalize hab hba
 
-@[simp] lemma dvd_normalize_iff {a b : α} : a ∣ normalize b ↔ a ∣ b :=
+lemma dvd_normalize_iff {a b : α} : a ∣ normalize b ↔ a ∣ b :=
```

Comment:
Can these now also be proven by simp? (You don't need to change the proof, since the current one is nice and fast.)
If so, it might make sense to add a comment `-- can be proven by simp` so that future edits will not try to put the `@[simp]` tag back.

Reply:
Yeah, that one-lemma proof is a simp lemma. I'll add those comments.